tests/unit/store/actions.spec.ts

```diff
@@ -180,6 +190,17 @@ describe('actions', () => {
       ])
       done()
     })
+
+    it('checks if rsql is still the same', async (done) => {
+      const commit = jest.fn()
+      const getters = { rsql: 'll_nr.yob=le=1970' }
+      const action = actions.loadGridData({ commit, getters })
+      expect(commit).toHaveBeenCalledWith('updateVariantCounts', [])
+      getters.rsql = ''
+      await action
+      expect(commit).toHaveBeenCalledTimes(1)
```

Comment:
same maybe at a note about how the called 1 time relates to the ‘rsql is still the same’

Reply:
I've updated the test description to make it clearer